docs/guides/server/tracing.adoc

```diff
@@ -17,9 +17,9 @@ It also provides valuable insights into performance bottlenecks and can help opt
 
 == Enable tracing
 
-It is possible to enable exposing traces using the build time option `tracing-enabled` as follows:
+It is possible to enable exposing traces by enabling the `opentelemetry` feature as follows:
 
-<@kc.start parameters="--tracing-enabled=true"/>
+<@kc.start parameters="--features=opentelemetry"/>
```

Comment:
My feeling is we should still primarily point users to the `tracing-enabled=true` option. It's a "sugar switch" for the feature and IMHO it's nicer than `features=opentelemetry`. WDYT @mabartos?